actions/workflows/st2_pkg_e2e_test.yaml

```diff
@@ -156,6 +156,32 @@ tasks:
     input:
       hosts: <% ctx().vm_info.private_ip_address %>
       timeout: 600
+    next:
+      - when: <% succeeded() %>
+        do: check_for_ubuntu_xenial
+  check_for_ubuntu_xenial:
+    action: core.remote
+    input:
+      hosts: <% ctx().vm_info.private_ip_address %>
+      cmd: grep 'Ubuntu 16' /etc/issue
+    next:
+      - when: <% succeeded() %>
+        do: install_python36
+      - when: <% failed() %>
+        do: check_debug_mode
+  install_python36:
+    action: core.remote_sudo
+    input:
+      hosts: <% ctx().vm_info.private_ip_address %>
+      cmd: |-
+        python3 --version
+        add-apt-repository ppa:deadsnakes/ppa --yes
+        apt-get update
+        apt-get install --yes python3.6
+        patch /etc/st2/st2.conf <<EOF
+        23a24
+        > python3_binary = /usr/bin/python3.6
```

Comment:
Yeah, with python3 only support in next st2 version, from what I understand we're also removing `python3_binary` from `st2.conf`

Reply:
I have a PR in WIP that will take out the --python3 option, and therfore the python3_binary parameter etc. It's just blocked at moment with the failures on the E2E - so I can't verify I've not broken anything.

Reply:
I reworked this PR to obviate this.